### ISSUE TITLE
配合QMUIKit实现APP内部控制暗黑模式

### ISSUE THREAD

**Issue by Snail-hash** [tigerAndBull/TABAnimated#173]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name          iOS
 Platform Version        15.0
 TABAnimated Version      5.0.0
 Xcode Version           XCode 13.0
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

APPDelegate设置TABAnimated的darkModeType 实际上还是跟随系统模式
[TABAnimated sharedAnimated].darkModeType = [QMUIThemeManagerCenter.defaultThemeManager.currentThemeIdentifier isEqual:QDThemeIdentifierDark] ? TABAnimatedDarkModeForceDark : TABAnimatedDarkModeForceNormal;
    if (@available(iOS 13.0, *)) {
        if (QMUIThemeManagerCenter.defaultThemeManager.respondsSystemStyleAutomatically) {
            [TABAnimated sharedAnimated].darkModeType = TABAnimatedDarkModeBySystem;
        }
    }

**Comment by tigerAndBull** [comment]
根据你的使用场景，是不能在appDelegate中设置的，应该在QMUI的darkMode被用户修改时，同时修改TABAnimated的darkMode
